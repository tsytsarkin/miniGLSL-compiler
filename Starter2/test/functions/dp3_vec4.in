{
    float f;
    vec4 v1;
    vec4 v2;
    f = dp3(v1, v2);
}
