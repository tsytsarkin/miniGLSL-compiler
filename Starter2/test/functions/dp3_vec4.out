  1: {
  2:     float f;
  3:     vec4 v1;
  4:     vec4 v2;
  5:     f = dp3(v1, v2);
  6: }
empty declarations
FLOAT_T type
declaration
declarations
VEC_T type
declaration
declarations
VEC_T type
declaration
declarations
empty statements
ID variable
function_name
ID variable
variable expression
ID variable
variable expression
arguments
function
function expression
assignment statement
statements
scope
program
